src/makeRobotAccountant.js

```diff
@@ -6,7 +6,15 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (n) => (n2) => {
+    counter++;
+
+    return counter % 2 === 0 && counter > 3
```

Comment:
It is better not to combine logical and ternary operators.
In this case you could assign `counter % 2 === 0 && counter > 3` to a variable.